src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+quarkus.log.category."com.vaadin.base.devserver.viteproxy".level=TRACE
+quarkus.vertx.event-loops-pool-size=2
```

Comment:
Is there any reason to limit the Vert.x event loop size?
Otherwise, I'd remove this setting and keep the default.

Reply:
These two were copied from the `main`. Maybe they were committed accidentially in some other commit?
If we are not sure, I can remove it from all places.

Reply:
On `v24` branch, the application.properties is empty, so I would do the same also for gradle